tests/data/dataset_readers/SQUAD_SAMPLE_FILE.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I tried putting the json in the test, but i found it to be non-trivial (since JSON doesn't allow line breaks in strings). Also, if you use the python `"""`, it keeps all of the spacing / I haven't found a function like scala's stripmargin. thoughts?